Simplified/NYPLBookCoverRegistry.m

```diff
@@ -85,6 +85,29 @@ - (NSURL *)URLForPinnedThumbnailImageOfBookIdentifier:(NSString *const)bookIdent
           URLByAppendingPathComponent:[bookIdentifier SHA256]];
 }
 
+- (void)coverImageForBook:(NYPLBook *)book handler:(void (^)(UIImage *image))handler
+{
+
+  //Thumbnail first as placeholder
+  [self thumbnailImageForBook:book handler:handler];
```

Comment:
This doesn't seem right to me. Given that `handler` will often have side effects, I think it would be very surprising for any user of this method to have their handler called multiple times, first with a thumbnail and later with a cover image.